docs/_documentations/eclipse-uninstall.md

```diff
@@ -9,19 +9,19 @@ type: document
 order: 70
 ---
 
-# Uninstalling Codewind from Eclipse
+# Uninstalling Codewind for Eclipse
 
-Remove the Codewind containers and images
+To uninstall Codewind for Eclipse, follow these instructions:
 
-1. Bring up the context menu of the **Codewind** item in the **Codewind Explorer**
-2. Click on **Uninstall**
-3. If your application containers are running, you will be prompted whether you want to stop them.
-
-Use the Eclipse uninstall support to uninstall Codewind for Eclipse:
+## Remove the Codewind containers and images
```

Comment:
Please can we number these instructions, so that it is clear that you first have to Remove the Codewind containers and images, and THEN uninstall codewind? i.e. you must perform both headered sets of instructions?
**1. Remove the codewind containers and images**
**2. Uninstall Codewind**

Reply:
The description for the eclipse topic needs fixing too, including formatting.

Reply:
Added numbering, and enhanced the descriptions for all three uninstalling topics.